src/pages/loading-waiting-states/spinner-overlays/data.tsx

```diff
@@ -0,0 +1,6 @@
+export const eulaDetails = {
```

Comment:
can we call this file `eulaText.tsx` or something more specific than just 'data'